### DISCUSSION TITLE
Is a new version coming soon?...

### DISCUSSION THREAD

**Discussion by Batnun** [chris2511/xca#432] [Q&A] [answered]
This is probably the best CA software I worked with, but XCA 2.4.0 was released almost a year and a half ago, is the project dead? :(

**Comment by ValentinLiu15** [comment]
I have the same question. There is a long way if XCA want to be a modern CA management. It need OCSP responder, embed SCT support, etc.

**Comment by FGTgb** [comment]
Same here, I would like to get the support of RSA-PSS certificates in a new version.

**Comment by teward** [comment]
@LeviMarvin I don't think XCA is designed to be a CA *server* in the sense of an OCSP responder.  At that point you're talking about a full PKI infrastructure, rather than a certificate manager, which is different from what I believe XCA was designed to be.

**Comment by bsbudhwar** [comment]
I am able to successfully deployed OCSP with **opensc-ocspd** on Debian 10 and 11 with XCA.

**Comment by spacexnasa** [comment]
actually,you can build yourself from sour code,it's easy

**Comment by AnonATXDO** [comment]
@chris2511 seems to be updating the repo recently more again. I assume, there's a release coming up to support OpenSSL 3 since the used OpenSSL 1.1.1 will no longer be supported as a LTS by the OpenSSL foundation soon. 

Of course, liking the software a lot, I wouldn't mind @chris2511 to give feedback or an estimate.

**Comment by Batnun** [comment]
This software is simply amazing, nothing is getting even close, I just can't wait for the new version! Thank you @chris2511 :)

Would love to have an option to donate or to sponsor this project, maybe this can pay for more development hours or resources...

**Comment by chris2511** [comment]
I'm working on it. I know it has been a long time since the last release. But if you are intrested, there are builds of all recent commits in the [Actions](https://github.com/chris2511/xca/actions).
They create installers (msi, dmg, portable.zip). You may use and test them on your own risk. I always try to keep the current `main` branch stable, but ....
Latest: https://github.com/chris2511/xca/actions/runs/5062501545

You are very welcome to test those builds and report any issues.
They just lack support for remote databases.
Here is the [changelog](https://github.com/chris2511/xca/blob/main/changelog)

**Reply by AnonATXDO** [reply]
Hi Chris, 
do all the builds lack remote database support in the Actions tab? I saw that a change of 2022 fixed the AES256 missing from the PKCS#12 file format and was wondering if anything in between does support a remote database plus this security feature. 
Thanks a lot!

**Comment by chris2511** [comment, marked as answer]
https://github.com/chris2511/xca/releases/tag/RELEASE.2.5.0

**Comment by Batnun** [comment]
Thanks Chris, much appreciated :)
